diagnostic.md

```diff
@@ -3,67 +3,59 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+#ajax-error-message, .container, .js-stale-session-flash, #facebox
 
 ## Question 2
 
 Using the diagram in the `README` file, what are two ways you could return ALL
 list items (`li`) using jQuery? This is intentionally vague. (Assume starting
 at the `OL` DOM node).
 
-<!-- your answer starts here -->
-
-<!-- your answer ends here -->
+jquery("li") and $("li")
```

Comment:
From `<ol>`, two possible ways to select `<li>` elements are `$('ol > li');` and `$('ol').children();`.  Some other possible ways are `$('li');`, `$('body').find('li');`, and `$('body').children('ol').children('li');`.